docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -368,14 +387,12 @@ means the document came from the local cluster.
 === Using async search for {ccs} with ccs_minimize_roundtrips=true
 
 Remote clusters can be queried asynchronously using the <<async-search,async search>> API.
-Async searches accept a <<ccs-minimize-roundtrips,`ccs_minimize_roundtrips`>> parameter
-that defaults to `false`. See <<ccs-min-roundtrips>> to learn more about this option.
+A {ccs} accepts a <<ccs-minimize-roundtrips,`ccs_minimize_roundtrips`>> parameter. For
+asynchronous searches it defaults to `false`. (Note: for synchronous searches it defaults to `true`.)
+See <<ccs-min-roundtrips>> to learn more about this option.
```

Comment:
Should we mention here that mrt=true typically would be expected to make the search faster, especially when there is network latency?

Reply:
Good idea. I'll add that to the top of the <ccs-min-roundtrips>> section.